noir-projects/noir-contracts/contracts/app/simple_token_contract/src/main.nr

```diff
@@ -89,7 +89,7 @@ pub contract SimpleToken {
     }
 
     #[utility]
-    pub(crate) unconstrained fn private_balance_of(owner: AztecAddress) -> u128 {
+    unconstrained fn private_balance_of(owner: AztecAddress) -> u128 {
```

Comment:
Pulled a Jan here: we no longer need for these functions to be `pub` as we no longer invoke them directly. Indeed we want to use https://github.com/noir-lang/noir/pull/9363 to prevent that, but we don't yet have this feature because of the noir sync issue.